frontend/app/src/components/FontFaceDeclaration/FontFaceDeclaration.tsx

```diff
@@ -27,15 +27,16 @@ const FontFaceDeclaration = ({
 }: FontFaceDeclarationProps): ReactElement => {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any -- TODO: Replace 'any' with a more specific type.
   const fontMarkup = fontFaces.map((font: any) => {
-    const { family, weight, url, style = "normal" } = font
-
+    const { family, weight, stringWeight, url, style, unicodeRange } = font
+    const resolvedWeight = weight || stringWeight
```

Comment:
nit: I think we might want to give `stringWeight` priority since its the newer option and `weight` is only the fallback for compatibility reasons: `stringWeight || weight`

Reply:
Maybe also add a comment here that the integer based `weight` is used as fallback here for compatibility reasons.